src/datasets/pings.md

```diff
@@ -31,3 +145,52 @@ As of September 28, 2018, members of the `meta` key on main pings include:
 <!-- table generated via `scripts/new_ping_metadata_table.py > src/cookbooks/new_ping_metadata_table.md` -->
 
 {{#include ../cookbooks/new_ping_metadata_table.md}}
+
+## Analysis
+
+The large majority of analyses can be completed using only the
+[main ping][main_ping].
+This ping includes histograms, scalars, and other performance and diagnostic data.
```

Comment:
```suggestion
The [main ping][main_ping] includes histograms, scalars, and other performance and diagnostic data.
Since Firefox 62, it [no longer contains event data](https://bugzilla.mozilla.org/show_bug.cgi?id=1460595); events are now sent in a separate `event` ping.
```